contrib/aws/provision-aws-cluster.sh

```diff
@@ -102,10 +110,12 @@ until [ `wc -w <<< $INSTANCE_STATUSES` -eq $DEIS_NUM_INSTANCES ]; do
     fi
 
     if [ $COUNTER -ne 1 ]; then sleep $SLEEPTIME; fi
-    echo "Waiting for instances to pass initial health checks ($(expr 61 - $COUNTER)0s) ..."
+    echo "Waiting for instances to pass initial health checks ($((61 - COUNTER))0s) ..."
+    # Don't warn about EXTRA_AWS_CLI_ARGS being unquoted
+    # shellcheck disable=SC2086
     INSTANCE_STATUSES=$(aws ec2 describe-instance-status \
         --filters Name=instance-status.reachability,Values=passed \
-        --instance-ids $INSTANCE_IDS \
+        --instance-ids "$INSTANCE_IDS" \
```

Comment:
This shouldn't be quoted: it makes all the instance IDs appear as one arg, which confused the provisioning script:
```
A client error (InvalidInstanceID.Malformed) occurred when calling the DescribeInstanceStatus operation: Invalid id: "i-661ddcaf
i-d4809923
i-ca80993d"
```